sudokuSolver/sudokuSolver/sudokuSolverFunctions.cpp

```diff
@@ -0,0 +1,174 @@
+#include "sudokuSolverFunctions.h"
+
+int findMissing(int a[], int b[], int n, int m)
+{
+    for (int i = 0; i < n; i++)
+    {
+        int j;
+        int counter=0;
+        for (j = 0; j < m; j++)
+        {
+            if (a[i] == b[j])
+            {
+                counter++;
+            }
+        }
+        if(counter==0)
+        {
+            return a[i];
+        }
+    }
+}
+
+bool zeroValidator(int a[])
+{
+    for (int i=0; i<9; i++)
+    {
+        if(a[i]==0)
+        {
+            return true;
```

Comment:
try to have only one return point in function

Reply:
done with :18cd8c9d48ab8dab4d1eac1c0c8076571383850b